template-manager-core/src/main/java/com/sequenceiq/cloudbreak/template/filesystem/CloudStorageConfigDetails.java

```diff
@@ -43,12 +43,7 @@ public Set<ConfigQueryEntry> queryParameters(BlueprintTextProcessor blueprintTex
                         .stream()
                         .filter(configQueryEntry -> configQueryEntry.getRelatedServices().stream().
                                 anyMatch(relatedService -> relatedService.equalsIgnoreCase(service)))
-                        .filter(configQueryEntry -> {
-                            if ((configQueryEntry.isRequiredForAttachedCluster() && attachedCluster) || !attachedCluster) {
-                                return true;
-                            }
-                            return false;
-                        })
+                        .filter(configQueryEntry -> configQueryEntry.isRequiredForAttachedCluster() || !attachedCluster)
```

Comment:
If the prevoius logic was right then it should be `configQueryEntry.isRequiredForAttachedCluster() && attachedCluster || !attachedCluster`